keyserver/handler.go

```diff
@@ -56,3 +65,12 @@ func (server *ConiksServer) handleRegistrationMessage(reg *p.RegistrationRequest
 		TB:   tb,
 	}, nil
 }
+
+func (server *ConiksServer) StoreUserPoliciesToKV(up *p.ConiksUserPolicies) error {
```

Comment:
I forgot to reply your comment :(

Do you think that the server still needs to know the user's policies in some cases, e.g., _signed/unsigned key change_ policy? If so, we have to store the user policies into db, right?